### DISCUSSION TITLE
Switch off xdebug once installed

### DISCUSSION THREAD

**Discussion by sta1r** [shivammathur/homebrew-extensions#4269] [General]
Hi - I've just today started using your excellent Homebrew tap and extensions! 

The way I used to run xDebug with PHP, I would simply comment out the extension reference in php.ini when needed, e.g.
```
zend_extension="\"
```
then re-comment it with a `;` to ensure PHP was as fast as possible. 

However - since I have installed the xDebug extension using
```
brew install shivammathur/extensions/xdebug@8.2
```
it's like the extension is now baked in. 

Is it possible to toggle it on and off without uninstalling the extension?

**Comment by shivammathur** [comment]
The ini files for extensions in the tap for PHP 8.2 would be here
```
cd "$(brew --prefix)"/etc/php/8.2/conf.d
```
